src/main/kotlin/com/criptext/mail/SecureEmail.kt

```diff
@@ -26,16 +26,16 @@ class SecureEmail {
         val LABEL_IMPORTANT = "IMPORTANT"
         val LABEL_CATEGORY_UPDATES = "CATEGORY_UPDATES"
         val LABEL_CATEGORY_FORUMS = "CATEGORY_FORUMS"
-        val LABEL_CHAT = "CHAT"
-        val LABEL_SENT = "SENT"
-        val LABEL_INBOX = "INBOX"
-        val LABEL_TRASH = "TRASH"
-        val LABEL_ALL_MAIL = "ALL MAIL"
+        val LABEL_CHAT = "Chat"
+        val LABEL_SENT = "Sent"
+        val LABEL_INBOX = "Inbox"
+        val LABEL_TRASH = "Trash"
+        val LABEL_ALL_MAIL = "All Mail"
         val LABEL_CATEGORY_PROMOTIONS = "CATEGORY_PROMOTIONS"
-        val LABEL_DRAFT = "DRAFT"
-        val LABEL_SPAM = "SPAM"
-        val LABEL_STARRED = "STARRED"
-        val LABEL_UNREAD = "UNREAD"
+        val LABEL_DRAFT = "Draft"
+        val LABEL_SPAM = "Spam"
+        val LABEL_STARRED = "Starred"
+        val LABEL_UNREAD = "Unread"
```

Comment:
Why use this instead of `Label.DefaultItems`?

Reply:
Now there are constants for the labels inside the label model companion object and the default label uses those instead, as well as all the code that needed to check for one of these.

Reply:
I would suggest to delete `SecureEmail.kt`